tide/core/DynamicTexture.cpp

```diff
@@ -235,19 +234,14 @@ bool DynamicTexture::writePyramidImagesRecursive( const QString& pyramidFolder )
     // recursively generate and save children images
     if( _canHaveChildren( ))
     {
-        QRectF imageBounds[4];
-        imageBounds[0] = QRectF( 0.0, 0.0, 0.5, 0.5 );
-        imageBounds[1] = QRectF( 0.5, 0.0, 0.5, 0.5 );
-        imageBounds[2] = QRectF( 0.5, 0.5, 0.5, 0.5 );
-        imageBounds[3] = QRectF( 0.0, 0.5, 0.5, 0.5 );
-
-#pragma omp parallel for
+        QList<DynamicTexturePtr> children;
         for( unsigned int i=0; i<4; ++i )
-        {
-            DynamicTexturePtr child( new DynamicTexture( "", shared_from_this(),
-                                                         imageBounds[i], i ));
+            children.push_back( DynamicTexturePtr(
```

Comment:
emplace_back( new DynamicTexture( shared_from_this(), i ) ) if you are C++11. Just nicer and without copy construction.